src/interface/console.c

```diff
@@ -737,7 +737,17 @@ static int cc_set(const utf8 **argv, int argc)
 		}
 
 		if (strcmp(argv[0], "money") == 0 && invalidArguments(&invalidArgs, double_valid[0])) {
-			gCashEncrypted = ENCRYPT_MONEY(MONEY((int)double_val[0], ((int)(double_val[0] * 100)) % 100));
+			money32 money = MONEY((int)double_val[0], ((int)(double_val[0] * 100)) % 100);
+			if (gCashEncrypted != ENCRYPT_MONEY(money)) {
+				if (game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETMONEY, money, GAME_COMMAND_CHEAT, 0, 0) != MONEY32_UNDEFINED) {
+					//Change it locally so it shows the accurate value in the 
+					//"console_execute_silent("get money")" line when in network client mode
+					gCashEncrypted = ENCRYPT_MONEY(money);
```

Comment:
Did you actually test it and prove that it will cause desyncs?

https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2/interface/console.c#L894
https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2/interface/console.c#L907
https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2/interface/console.c#L920
https://github.com/OpenRCT2/OpenRCT2/pull/4672

It has already been done before, it's being used in those lines, and was discussed in the linked pull request. Unless @Gymnasiast or @IntelOrca agree with you, I am keeping it. This pull request was already snipped from a previous pull request because that was too much to handle, so a callback system should be put in a seperate pull request.